runtime/compiler/trj9/ilgen/IlGenerator.cpp

```diff
@@ -1484,10 +1484,9 @@ TR_J9ByteCodeIlGenerator::genHWOptimizedStrProcessingAvailable()
    if (constToLoad == -1)
       {
       bool disable = comp()->getOption(TR_DisableSIMDStringCaseConv);
-      bool platformSupported = TR::Compiler->target.cpu.isZ();
-      bool vecInstrAvailable = cg()->getSupportsVectorRegisters();
+      bool platformSupported = cg()->getSupportsSIMDStringCaseConv(); 
```

Comment:
This variable has exactly one occurrence. Can we inline its use? Similarly to below we can get rid of the `!disable` check and fold that variable away.